ui/ui.js

```diff
@@ -131,6 +131,11 @@ shaka.ui.Overlay = class {
 
     goog.asserts.assert(this.player_ != null, 'Should have a player!');
 
+    // Check AirPlay support
+    if (window.WebKitPlaybackTargetAvailabilityEvent) {
+      this.config_.overflowMenuButtons.push('airplay');
```

Comment:
Wait, this automatically adds the AirPlay button to the config whenever you call `configure`? What happens if the developer doesn't WANT an AirPlay button? I feel like this should go inside `defaultConfig_` instead.

Reply:
I changed it!